scripts/build.sh

```diff
@@ -3,7 +3,7 @@
 npm run dist
 
 # Only run E2E examples on Node 6+
-if [[ "${TRAVIS_NODE_VERSION}" =~ ^(6|7|8|9|10) ]]; then
+if [[ "${TRAVIS_NODE_VERSION}" =~ ^(6|7|8|9)|1[0-9]+ ]]; then
```

Comment:
It would be neat if we could pull this logic outside of the two platform specific scripts - maybe with  [`check-node-version`](https://www.npmjs.com/package/check-node-version) - which also would mean the script keeps working when we get to node 20.